src/devices/Tca954x/README.md

```diff
@@ -0,0 +1,115 @@
+# TCA954X - TCA954X Low-Voltage Multi-Channel I2C Switch with Reset
+
+The TCA954X/PCA954X group of devices is a set of I2C switches. Several types are available, with 2, 4 or 8 channels, and different voltage specifications. The first letter is sometimes also "P". Not to be confused with PCA95x4, which is a set of GPIO expanders.
+
+The TCA9548A device has eight bidirectional translating switches that can be controlled through the I2C bus. The SCL/SDA upstream pair fans out to eight downstream pairs, or channels.
+
+All operations for basic usage has been developed. If needed, RESET operation can also be implemented.
+
+## Documentation
+
+![TCA9548A](./TCA9548A.jpg)
+
+TCA9548A [datasheet](https://www.ti.com/lit/ds/symlink/tca9548a.pdf)
+
+You will find this device as ["Adafruit TCA9548A 1-to-8 I2C Multiplexer Breakout"](https://learn.adafruit.com/adafruit-tca9548a-1-to-8-i2c-multiplexer-breakout)
+
+A list of [different variants](https://www.ti.com/interface/i2c/switches-and-multiplexers/products.html?keyMatch=TCA9548) is available on the Texas Instruments Website.
+
+## Usage
+
+Create a ```Tca9548A``` class and pass the I2C device. The default I2C address is provided in the class.
+
+```csharp
+            Console.WriteLine("Hello TCA9548A!");
```

Comment:
detail, but you have everything on the left properly inented?

Reply:
Yea, indentation seems right.